clang/test/Driver/fp-model.c

```diff
@@ -75,6 +76,7 @@
 // RUN:   --check-prefix=WARN12 %s
 // RUN: %clang -### -ffast-math -ffp-model=strict -c %s 2>&1 | FileCheck \
 // RUN:   --check-prefix=WARN12 %s
+// WARN12: clang
```

Comment:
`clang` is unneeded. For diagnostics or -cc1 options, we just avoid check `clang` before `-cc1` since `clang` carries very little weight

Reply:
The only purpose of the check here is to establish a reference point for the WARN12-NOT check on the following line. I can change the thing I'm checking to "-cc1" if that will cause less confusion for people reading the test.